Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -185,14 +184,7 @@ def addSeeding(
         random number generator. Only used by SeedingAlgorithm.TruthSmeared.
     """
 
-    def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
-        """override logging level"""
-        if logLevel is None:
-            return s.config.logLevel
-        return acts.logging.Level(max(custom.value, logLevel.value))
-
-    if int(customLogLevel()) <= int(acts.logging.DEBUG):
-        acts.examples.dump_args_calls(locals())
```

Comment:
As discussed previously, `dump_args_calls` is no longer called with `DEBUG` `logLevel`. I think this is the only change to existing functionality.
Please comment on this in the PR description.